doc/htmldoc/model_details/aeif_models_implementation.ipynb

```diff
@@ -75,19 +76,9 @@
     "\n",
     "* [numpy](http://www.numpy.org/) and [scipy](http://www.scipy.org/)\n",
     "* [assimulo](http://www.jmodelica.org/assimulo)\n",
-    "* [matplotlib](http://matplotlib.org/)\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Install assimulo package in the current Jupyter kernel\n",
-    "import sys\n",
+    "* [matplotlib](http://matplotlib.org/)\n",
     "\n",
-    "!{sys.executable} -m pip install assimulo"
+    "Note that the last version of Assimulo available from PyPI is version 3.0, which is not compatible with current versions of Python distribution tools. If you use conda/mamba, you can install a current version of Assimulo from `conda-forge`."
```

Comment:
```suggestion
    "Note that the latest version of Assimulo available from PyPI is version 3.0, which is not compatible with current versions of Python distribution tools. If you use conda/mamba, you can install a current version of Assimulo from `conda-forge`."
```

Reply:
Also can we mention which minimal version of Assimulo works?

Reply:
@jessica-mitchell I revised these two sentences to achieve greater clarity.